priority_variables_transform/CHS-ingest/asthma.yaml

```diff
@@ -473,6 +474,29 @@
         relationship_to_participant:
           value: ONESELF
           range: string
+- class_derivations:
+    Condition:
+      populated_from: pht001490
+      slot_derivations:
+        associated_participant:
+          populated_from: phv00105099
+        associated_visit:
+          value: CHS YEAR 5 NEW
+          range: string
+        age_at_condition_start:
+          expr: case(phv00106022 == 1, {phv00106025} * 365)
+        age_at_condition_end:
+          expr: case(phv00106022 == 1, (phv00106023 == 0, {phv00106026} * 365))
```

Comment:
Here too, but this one is also missing an internal case:
```
expr: case((phv00106022 == 1, case((phv00106023 == 0, {phv00106026} * 365))))
```

Reply:
fixed